modules/tinymce/src/plugins/table/test/ts/module/test/TableModifiersTestUtils.ts

```diff
@@ -8,12 +8,16 @@ import Editor from 'tinymce/core/api/Editor';
 interface AssertStyleOptions {
   menuTitle: string;
   subMenuTitle: string;
-  checkMarkEntries: number;
+  subMenuTitleUndo: string;
```

Comment:
Undo isn't really an accurate name here, as it's removing the style not undoing it. So maybe `subMenuRemoveTitle` instead?

Reply:
Sure.